babel.config.js

```diff
@@ -4,5 +4,10 @@ module.exports = function( api ) {
 	return {
 		presets: [ '@wordpress/babel-preset-default' ],
 		plugins: [ 'babel-plugin-inline-json-import' ],
+		env: {
+			test: {
```

Comment:
It's already scoped to stories with the current setup:
- it is scoped only for Storybook because the Babel config is local for the tool
- it gets activated only when running tests

I don't think there is any difference between those two options and they also don't share any preference in their docs. In addition to that, macros struggle with the cache invalidation for the dependent files. See the comment in `create-react-app` https://github.com/facebook/create-react-app/commit/11737bc78676868e76ce5d0c04ddbb5c758d3908#diff-b1524f1e1ccabdb327a12f23b432b20bR13: 

> 
>     // Babel Macros are notoriously hard to cache, so they shouldn't be
>     // https://github.com/babel/babel/issues/8497
>     // We naively detect macros using their package suffix and insert a random
>     // caller name, a valid option accepted by Babel, to compose a one-time
>     // cacheIdentifier for the file. We cannot tune the loader options on a per
>     // file basis.

Reply:
That makes sense, thanks for clarifying.